### PR TITLE
Add environment variable for cluster waiting time

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
Resolves https://github.com/wso2-cellery/sdk/issues/833

### CODE REVIEW

File: components/cli/pkg/runtime/runtime.go [hunk 1]
```diff
@@ -461,12 +463,37 @@ func WaitFor(checkKnative, hpaEnabled bool) {
...
 	spinner.Stop(true)
+}
 
+func waitingTimeCluster() time.Duration {
+	waitingTime := time.Hour
+	envVar := os.Getenv("CELLERY_CLUSTER_WAIT_TIME")
+	if envVar != "" {
+		wt, err := strconv.Atoi(envVar)
+		if err != nil {
+			util.ExitWithErrorMessage("Error getting waiting time", err)
```
Comment (isurulucky) [inline components/cli/pkg/runtime/runtime.go]: Return the error and handle from parent.

File: components/cli/pkg/runtime/runtime.go [hunk 2]
```diff
@@ -461,12 +463,37 @@ func WaitFor(checkKnative, hpaEnabled bool) {
...
+	return waitingTime
+}
+
+func waitingTimeCellerySystem() time.Duration {
+	waitingTime := time.Minute*15
+	envVar := os.Getenv("CELLERY_SYSTEM_WAIT_TIME")
+	if envVar != "" {
+		wt, err := strconv.Atoi(envVar)
+		if err != nil {
+			util.ExitWithErrorMessage("Error getting waiting time", err)
```
Comment (isurulucky) [inline components/cli/pkg/runtime/runtime.go]: Return the error and handle from parent.

Comment (MadhukaHarith92) [reply components/cli/pkg/runtime/runtime.go]: Update.

File: components/cli/pkg/runtime/runtime.go (hunk 1 above)
Comment (MadhukaHarith92) [reply components/cli/pkg/runtime/runtime.go]: Updated.

**Pull request merged**